.gitignore

```diff
@@ -1 +1,3 @@
 target/
+.bsp/
+.idea/
```

Comment:
I would expect individuals to manage these in their own global gitignore files since they'll depend on their choice of IDE and other tooling.

Reply:
Ah that's a good idea! I just had to do it quickly so that sbt-release would work. Have now amended